pandas/io/gbq.py

```diff
@@ -37,6 +39,10 @@ def _check_google_client_version():
 logger = logging.getLogger('pandas.io.gbq')
 logger.setLevel(logging.ERROR)
 
+def _print(msg, end='\n'):
+	sys.stdout.write(msg + end)
```

Comment:
you should pass ``verbose`` to this (and do the if inside here). That way you can simply:

``_print(msg, verbose=verbose)`` in the code itself.